### DISCUSSION TITLE
Seperate Forked Respository created by a user from the Respositories owned by a user

### DISCUSSION THREAD

**Discussion by hslee2008** [community/community#48244] [Repositories]
### Select Topic Area

Product Feedback

### Body

I think most people would agree that it might be better to seperate Forked Repos from Repos owned by the user.

# Perspective of the visitor
They would probably want to look at repos originally made by a user.

Or they might want to see what the user is working on: in open source (forked repos).

They might get confused with the forked repos because the label for the forked repos are very hard to discern between lists of repos.

# Perspective of the user
From experience, I want to look at repositories I am currently contributing to (like forked repos) and repositories I am working on myself (owned by me).

It will also be more organized.

# Solution
Maybe create a seperate tab for forked repos 

or 

create two sections so that users can look at each category one by one instead of mixing them both.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @HyunseungLee-Travis! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
